lib/nrf_modem_lib/nrf_modem_os.c

```diff
@@ -230,8 +230,7 @@ bool nrf_modem_os_is_in_isr(void)
 	return k_is_in_isr();
 }
 
-#define NRF_MODEM_OS_SEM_MAX 3
-static struct k_sem nrf_modem_os_sems[NRF_MODEM_OS_SEM_MAX];
+static struct k_sem nrf_modem_os_sems[NRF_MODEM_OS_NUM_SEM_REQUIRED];
```

Comment:
Are the changes in `nrf_modem_os.c` related to the lwm2m_carrier update?

Reply:
We were just including the commit(s) from libmodem PR in our PR until the libmodem PR got merged. Rebased not that løibmodem v1.5.0 has been merged.